src/server/scripts/Northrend/Ulduar/Ulduar/boss_mimiron.cpp

```diff
@@ -2187,8 +2187,8 @@ class spell_mimiron_rocket_strike : public SpellScriptLoader
                 if (targets.empty())
                     return;
 
-                if (m_scriptSpellId == SPELL_ROCKET_STRIKE_LEFT && GetCaster()->IsVehicle())
-                    if (WorldObject* target = GetCaster()->GetVehicleKit()->GetPassenger(6))
+                if (m_scriptSpellId == SPELL_ROCKET_STRIKE_SINGLE && GetCaster()->IsVehicle())
+                    if (WorldObject* target = GetCaster()->GetVehicleKit()->GetPassenger(5 + irand(0, 1)))
```

Comment:
can you add an enum with useful seatids, then you can use here something like RAND(SEAT_ROCKET_1, SEAT_ROCKET_2) for more legibility